server/src/main/java/org/opensearch/gateway/GatewayModule.java

```diff
@@ -47,6 +47,7 @@ protected void configure() {
         bind(GatewayService.class).asEagerSingleton();
         bind(TransportNodesListGatewayMetaState.class).asEagerSingleton();
         bind(TransportNodesListGatewayStartedShards.class).asEagerSingleton();
+        bind(TransportNodesListGatewayStartedBatchShards.class).asEagerSingleton();
```

Comment:
`TransportNodesListGatewayStartedShardsBatch` let's use the same naming convention as followed in https://github.com/opensearch-project/OpenSearch/pull/8742/files

Reply:
Pushed a commit renaming the file to `TransportNodesListGatewayStartedShardsBatch`